lib/facter/operatingsystem/linux.rb

```diff
@@ -247,7 +251,6 @@ def get_operatingsystem_with_release_files
           "AristaEOS"   => "/etc/Eos-release",
           "Debian"      => "/etc/debian_version",
           "Gentoo"      => "/etc/gentoo-release",
-          "Fedora"      => "/etc/fedora-release",
```

Comment:
Is this intended?

Reply:
What do you mean international change?

Reply:
What is the purpose of the Fedora-related changes?

Reply:
Fedora related change was done because all of ALT linux releases have fedora-release, redhat-release files in /etc, and since the code determines the os by the table firstly it will detect the OS as Fedora not as the ALT, so Fedora line was cropped out of the table, and inserted below the ALT Linux detection.

I've updated the PR, along the slightly changed and validated return strings.

Reply:
Thanks for the clarification, thought that might be the case :).